packages/server-wallet/src/protocols/channel-closer.ts

```diff
@@ -42,6 +42,10 @@ export class ChannelCloser {
       if (!channel) {
         throw new Error('Channel must exist');
       }
+
+      await channel.$fetchGraph('funding', {transaction: tx});
```

Comment:
Am I right to understand that we are changing the `channel-closer` implementation here because of the `directly-funded-channel` test?

Or, are you instead saying that
> Actually, it's important that we fetch the `funding` relationship here, as it's an important part of the `closeChannel` api

Reply:
This is a good question, and I do not know the answer. We are **not** changing the behavior of `channel-closer`. The channel-closer protocol state (which is now removed) used to fetch `funding` and `chainServiceRequests` for us via `store.getChannelState`.

Reply:
Ah, I understand now.
1. The refactor unintentionally changed the behaviour
2. https://github.com/statechannels/statechannels/pull/3249/commits/57a759ceaee22302b97158bf20e161aff869b210 reverted that behaviour change

Reply:
Yep. I probably should have added that to the commit message.

Reply:
If you squash & merge, the context can be included in its commit message